rele/middleware.py

```diff
@@ -47,6 +47,11 @@ def post_publish(self, topic):
         :param topic:
         """
 
+    def post_publish_failure(self, topic):
+        """Called after Publisher fails.
+        :param topic:
```

Comment:
Interesting. I wonder if its worth to pass in the Timeout exception to the hook, so that way we can either silence the exception, or raise with the original stack trace. What do you think?

Reply:
Actually I think we should yeah. I've noticed we also pass the exception in `post_process_message_failure`.